### PR TITLE
Issue 14339 integration tests

### PR DESCRIPTION
**Pull request by fabrizzio-dotCMS**

### CODE REVIEW

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowResourceIntegrationTest.java [hunk 1]
```diff
@@ -652,4 +712,559 @@ public void testDeleteScheme() throws Exception{
...
+        assertTrue(documentActions.stream()
+                .anyMatch(action -> SEND_TO_LEGAL.equals(action.getName())));
+        assertTrue(documentActions.stream().anyMatch(action -> PUBLISH.equals(action.getName())));
+        assertTrue(
+                documentActions.stream().anyMatch(action -> REPUBLISH.equals(action.getName())));
+        assertTrue(
+                documentActions.stream().anyMatch(action -> UNPUBLISH.equals(action.getName())));
+        assertTrue(documentActions.stream().anyMatch(action -> ARCHIVE.equals(action.getName())));
+        assertTrue(documentActions.stream()
+                .anyMatch(action -> TWEET_THIS.equals(action.getName())));
```
Comment (jgambarios) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowResourceIntegrationTest.java]: We can not depend of the Tweeter Action or Actionlet as will be remove it from the starter.

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowTestUtil.java [hunk 2]
```diff
@@ -46,6 +49,20 @@
...
+    static final String SEND_FOR_REVIEW = "Send for Review";
+    static final String SEND_TO_LEGAL = "Send to Legal";
+    static final String COPY = "Copy";
+    static final String SAVE = "Save";
+    static final String PUBLISH = "Publish";
+    static final String REPUBLISH = "Republish";
+    static final String UNPUBLISH = "Unpublish";
+    static final String ARCHIVE = "Archive";
+    static final String SAVE_PUBLISH = "Save / Publish";
+    static final String TWEET_THIS = "Tweet This!";
```
Comment (jgambarios) [inline dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowTestUtil.java]: Same as above, we can not depend of the Tweeter Action or Actionlet as will be remove it from the starter.

**Review by jgambarios — CHANGES_REQUESTED** [review]

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowResourceIntegrationTest.java (hunk 1 above)
Comment (fabrizzio-dotCMS) [reply dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowResourceIntegrationTest.java]: removed.

File: dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowTestUtil.java (hunk 2 above)
Comment (fabrizzio-dotCMS) [reply dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/workflow/WorkflowTestUtil.java]: removed.

**Review by jgambarios — APPROVED** [review]

**Review by jgambarios — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/concurrent/DotConcurrentFactory.java [hunk 3]
```diff
@@ -58,6 +70,7 @@
 
     public static final String DOT_SYSTEM_THREAD_POOL = "dotSystemPool";
 
+    public static final String BULK_ACTIONS_THREAD_POOL = "bulkActionsPool";
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/concurrent/DotConcurrentFactory.java]: I think this should be in the WorkflowAPI

File: dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java [hunk 4]
```diff
@@ -282,20 +282,25 @@ public final Response getBulkActions(@Context final HttpServletRequest request,
...
-                            fireBulkActionsForm,
-                            initDataObject.getUser()
-                    ), asyncResponse
-            );
+            ResponseUtil.handleAsyncResponse(() -> {
+                try {
+                    final BulkActionsResultView view = workflowHelper
+                            .fireBulkActions(fireBulkActionsForm, initDataObject.getUser());
+                    return Response.ok( new ResponseEntityView(view)).build();
+                } catch (Exception e) {
```
Comment (jdotcms) [inline dotCMS/src/main/java/com/dotcms/rest/api/v1/workflow/WorkflowResource.java]: you should log it

**Pull request merged**